x-pack/plugins/security_solution/public/management/state/async_resource_builders.ts

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
What is the purpose of this?

Reply:
I'm using them currently in tests, but will also be using them in the middleware in order to build the AsyncResourceState needed for `dispatch()`

Reply:
Should it be the same than `x-pack/plugins/security_solution/public/management/state/async_resource_state.ts`?